python/cuml/model_selection/__init__.py

```diff
@@ -1,4 +1,28 @@
+#
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
 from cuml.model_selection._split import train_test_split
+from sklearn.model_selection import GridSearchCV
+
 
+GridSearchCV.__doc__ = """
```

Comment:
ah, bummer that we have to copy past since these are in separate files (pipeline and model_selection) - if it goes to a third file ever I think we should break it out in a shared way